spec/rspec/core/formatters/snippet_extractor_spec.rb

```diff
@@ -250,6 +250,24 @@ def another_expression(*)
         end
       end
 
+      context 'when Ripper cannot parse the source', :isolated_directory do
+        let(:file_path) do
+          'invalid_source.rb'
+        end
+
+        let(:line_number) do
+          1
+        end
```

Comment:
Recently I tend to use consistent block style in all contexts, though I won't insist.